tgui/.eslintrc.yml

```diff
@@ -651,7 +651,7 @@ rules:
   ## Enforce ES5 or ES6 class for React Components
   react/prefer-es6-class: error
   ## Enforce that props are read-only
-  react/prefer-read-only-props: error
+  # react/prefer-read-only-props: error
```

Comment:
we don't do this anywhere and with the newer versioning it's detecting it and attempting to autofix. it does this terribly by appending `readonly` to every type, which could instead be wrapped by `Readonly<x>`